### PR TITLE
Refine struct define for Bybit

### PR DESCRIPTION
**Pull request by sunlei**
# Pull Request

Refine struct define for Bybit.

## Type of change

Delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: nautilus_trader/adapters/bybit/endpoints/trade/batch_place_order.py [hunk 1]
```diff
@@ -29,16 +29,21 @@
 from nautilus_trader.core.nautilus_pyo3 import HttpMethod
 
 
-class BybitBatchPlaceOrder(msgspec.Struct, omit_defaults=True, frozen=True):
+class BybitBatchPlaceOrder(
```
Comment (cjdsellers) [inline nautilus_trader/adapters/bybit/endpoints/trade/batch_place_order.py:32]: Do we intend on using all of these fields? or was the goal here to exactly match the API spec?

File: nautilus_trader/adapters/bybit/endpoints/trade/place_order.py [hunk 2]
```diff
@@ -16,43 +16,21 @@
...
-from nautilus_trader.adapters.bybit.common.enums import BybitTriggerType
 from nautilus_trader.adapters.bybit.endpoints.endpoint import BybitHttpEndpoint
+from nautilus_trader.adapters.bybit.endpoints.trade.batch_place_order import BybitBatchPlaceOrder
 from nautilus_trader.adapters.bybit.http.client import BybitHttpClient
 from nautilus_trader.adapters.bybit.schemas.order import BybitPlaceOrderResponse
 from nautilus_trader.core.nautilus_pyo3 import HttpMethod
 
 
-class BybitPlaceOrderPostParams(msgspec.Struct, omit_defaults=True, frozen=True):
+class BybitPlaceOrderPostParams(
```
Comment (cjdsellers) [inline nautilus_trader/adapters/bybit/endpoints/trade/place_order.py:27]: Is this the right inheritance direction, or its only `BybitPlaceOrder` which needs the `category`?

**Review by cjdsellers — APPROVED** [review]

File: nautilus_trader/adapters/bybit/endpoints/trade/batch_place_order.py (hunk 1 above)
Comment (sunlei) [reply nautilus_trader/adapters/bybit/endpoints/trade/batch_place_order.py:32]: I compared the fields one by one with those in the documentation.

https://bybit-exchange.github.io/docs/v5/order/create-order

File: nautilus_trader/adapters/bybit/endpoints/trade/place_order.py (hunk 2 above)
Comment (sunlei) [reply nautilus_trader/adapters/bybit/endpoints/trade/place_order.py:27]: `BybitPlaceOrderPostParams` has `.category`.

`BybitBatchPlaceOrderPostParams` has `.category` and `.request`.

**Pull request merged**